erpnext/projects/doctype/timesheet/timesheet.py

```diff
@@ -194,7 +194,7 @@ def set_to_time(self, data):
 			return
 
 		_to_time = get_datetime(add_to_date(data.from_time, hours=data.hours, as_datetime=True))
-		if data.to_time != _to_time:
+		if time_diff_in_seconds(_to_time, data.to_time) > 1:
```

Comment:
```suggestion
		if abs(time_diff_in_seconds(_to_time, data.to_time)) > 1:
```

Can also change >1 second in the other direction, I guess? (Same applies to `develop` PR).

Also, shouldn't it be `>=`? Otherwise we'd ignore 1 second changes.